### DISCUSSION TITLE
file picker is not working with live preview

### DISCUSSION THREAD

**Discussion by kangaroolab** [phcode-dev/phoenix#1844] [General]
Hi I tried to use file picker to save to local file but it is not working with live preview due to following error:

Error saving file: DOMException: Failed to execute 'showSaveFilePicker' on 'Window': Cross origin sub frames aren't allowed to show a file picker.

how could this work with phcode?

**Comment by abose** [comment]
Thanks for reporting the issue. This is not doable currently as the live preview is an iframe and hence is blocked by the browser. The only workaround for the time being is to popout the live preview.

We may be able to fix this in the future when we port to a newer Tauri-2 framework, but that is not planned within this year. Created an issue to track this https://github.com/phcode-dev/phoenix/issues/1849
